packages/styletron-react/src/styled.js

```diff
@@ -6,6 +6,18 @@ const isValidAttr = require('./is-valid-attr');
 const STYLES_KEY = '__STYLETRON_STYLES';
 const TAG_KEY = '__STYLETRON_TAG';
 
+if (typeof window !== 'undefined' && !window.__STYLETRON_DEVTOOLS__) {
```

Comment:
Maybe these new blocks should be wrapped with `if (PROCESS.ENV.NODE_ENV !== 'production') { ... }`?